src/main/java/mate/academy/springbootintro/model/User.java

```diff
@@ -26,4 +34,47 @@ public class User {
     @Column(nullable = false)
     private String lastName;
     private String shippingAddress;
```

Comment:
Think about how we should add columns like is_deleted. Should we add them directly to existing scripts or there is another better way?